cluster-autoscaler/utils/taints/taints.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package taints
+
+import (
+	"strings"
+
+	apiv1 "k8s.io/api/core/v1"
+	coreUtils "k8s.io/autoscaler/cluster-autoscaler/core/utils"
```

Comment:
I'd like to avoid importing core from utils as much as possible (and here it seems easy to do). We've had multiple problems in the past with various utils importing core and creating import cycles when we tried to use them somewhere else in the code.